src/hazelcore/CursorInfo_common.re

```diff
@@ -13,7 +27,7 @@ type typed =
   /* cursor in analytic position */
   | AnaAnnotatedLambda(HTyp.t, HTyp.t)
   // cursor is on a type inconsistent expression
-  | AnaTypeInconsistent(HTyp.t, HTyp.t)
+  | AnaTypeInconsistent(HTyp.t, HTyp.t, varexp)
```

Comment:
What is `varexp` and what are the reasons for adding it to some constructors vs others?

Reply:
For those cursor position where variable expressions may appear, I add `varexp` to its constructor.